Makefile

```diff
@@ -38,18 +38,21 @@ test: binaries/client
 
 .PHONY: integration-linux
 integration-linux: build-extension-types build-extensions
+	-find ~/.config/Logs/Lens -type f -name *.log -delete
```

Comment:
These are probably debug leftovers?

Reply:
I sort of wanted them to help with debugging in the future, but I'll open a separate PR for that after 4.1